charts/nginx-gateway-fabric/templates/certs-job.yaml

```diff
@@ -153,4 +153,4 @@ spec:
       securityContext:
         fsGroup: 1001
         runAsNonRoot: true
-  ttlSecondsAfterFinished: 0
+  ttlSecondsAfterFinished: 90
```

Comment:
Let's make this a configurable field, because different use cases may have different needs.